src/snowflake/connector/network.py

```diff
@@ -595,7 +595,7 @@ def _token_request(self, request_type):
                 },
             )
 
-    def _heartbeat(self) -> None:
+    def _heartbeat(self) -> Any | dict[Any, Any] | dict[str, Any] | None:
```

Comment:
`dict[Any, Any]` includes `dict[str, Any]`. No need to have both!
```suggestion
    def _heartbeat(self) -> Any | dict[Any, Any] | None:
```